scanners/git-repo-scanner/parser/__testFiles__/git-scanner-test-findings.json

```diff
@@ -0,0 +1,542 @@
+[
+  {
+    "name": "Github Repo",
+    "description": "A Github repository",
+    "category": "Git Repository",
+    "osi_layer": "APPLICATION",
+    "severity": "INFORMATIONAL",
+    "attributes": {
+      "id": 80711933,
+      "web_url": "https://api.github.com/repos/secureCodeBox/secureCodeBox",
```

Comment:
Is this really correct with api.github.com?
I'd assume that gitleaks could not handle this.

Reply:
Yes!! Thank you 👍